src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageAction.java

```diff
@@ -117,10 +117,20 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                                                    && Const.FeedbackSessionResults.QUESTION_SORT_TYPE.equals(sortType);
         boolean isShowSectionWarningForParticipantView = !data.getBundle().isComplete
                                                    && !Const.FeedbackSessionResults.QUESTION_SORT_TYPE.equals(sortType);
+        
+        // Warning for section wise does not make sense if there are no multiple
+        // sections.
+        boolean isMultipleSectionAvaialble = data.getBundle().getRosterSectionTeamNameTable().size() > 1;
+
         if (selectedSection.equals(ALL_SECTION_OPTION) && (isShowSectionWarningForParticipantView
-                                                           || isShowSectionWarningForQuestionView)) {
-            statusToUser.add(new StatusMessage(Const.StatusMessages.FEEDBACK_RESULTS_SECTIONVIEWWARNING,
-                                               StatusMessageColor.WARNING));
+                || isShowSectionWarningForQuestionView)) {
+            if (isMultipleSectionAvaialble) {
+                statusToUser.add(new StatusMessage(Const.StatusMessages.FEEDBACK_RESULTS_SECTIONVIEWWARNING,
+                        StatusMessageColor.WARNING));
```

Comment:
Align `[S]tatusMessageColor.WARNING` with `[C]onst.StatusMessages` like how it was done before.